wled00/network.cpp

```diff
@@ -402,8 +402,8 @@ void WiFiEvent(WiFiEvent_t event)
         ETH.config(INADDR_NONE, INADDR_NONE, INADDR_NONE);
       }
       // convert the "serverDescription" into a valid DNS hostname (alphanumeric)
-      char hostname[64];
-      prepareHostname(hostname);
+      char hostname[33];
```

Comment:
Why the change from 64 to 33 and this seems out of step with other parts of the code where you are using 25

Reply:
None of the UI fields are 64 bytes.